api/client/gro_client.py

```diff
@@ -62,6 +62,8 @@ def get_df(self):
         """
         while self._data_series_queue:
             data_series = self._data_series_queue.pop()
```

Comment:
do we want to cover the cases that the original input entity has `show_revisions=True`?

Reply:
yes for the user who wants ot directly control the details, you can put show_revisions on individual data series  via add_single_data_series directly. but this at get_df level, gives a shortcut to just do it on all series.